powersimdata/input/converter/tests/test_pypsa_to_grid.py

```diff
@@ -28,8 +28,11 @@ def test_import_network_including_storages_from_pypsa_to_grid():
     n = pypsa.examples.storage_hvdc()
     grid = FromPyPSA(n)
 
+    inflow = n.get_switchable_as_dense("StorageUnit", "inflow")
+    has_inflow = inflow.any()
+
     assert not grid.bus.empty
-    assert len(n.buses) == len(grid.bus)
+    assert len(n.buses) + has_inflow.sum() == len(grid.bus)
```

Comment:
Perhaps we could add test on the `plant` data frame:
```
assert len(n.generators) + has_inflow.sum() == len(grid.plant)
assert all(["inflow" in i for i in grid.plant.iloc[len(grid.plant) - has_inflow.sum():].index])
```

Reply:
yes1

Reply:
Is fixed now. "Unresolve" if there is still something missing.